packages/components/README.md

```diff
@@ -29,17 +29,14 @@ Many components include CSS to add styles, which you will need to load in order
 
 In non-WordPress projects, link to the `build-style/style.css` file directly, it is located at `node_modules/@wordpress/components/build-style/style.css`.
 
-### Popovers and Tooltips
+### Popovers
 
-_If you're using [`Popover`](/packages/components/src/popover/README.md) or [`Tooltip`](/packages/components/src/tooltip/README.md) components outside of the editor, make sure they are rendered within a `SlotFillProvider` and with a `Popover.Slot` somewhere up the element tree._
+_If you're using [`Popover`](/packages/components/src/popover/README.md) outside of the editor, make sure they are rendered within a `SlotFillProvider` and with a `Popover.Slot` somewhere up the element tree._
```

Comment:
I wonder if this paragraph can also be removed, after @youknowriad's recent `Popover` updates (for reference:

- https://github.com/WordPress/gutenberg/pull/53889
- https://github.com/WordPress/gutenberg/pull/53907
- https://github.com/WordPress/gutenberg/pull/53982
- https://github.com/WordPress/gutenberg/pull/54022

Reply:
Riad, I guess this paragraph is out of date and can be deleted?

Reply:
Yes, it's out of date. The Provider is no longer mandatory but you can still use it with the Slot if you want to control the exact position of the popover.